plugins/cilium-cni/cilium-cni.go

```diff
@@ -515,6 +515,11 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 		Sandbox: args.Netns,
 	})
 
+	// Add to the result the Interface as index of Interfaces
+	for i := range res.Interfaces {
```

Comment:
AFAIU the binding between `Interfaces` and `IPs` is positional so this should be enough.